server/apps/publish/archive_publish.py

```diff
@@ -342,12 +348,24 @@ def publish(self, doc, updates, target_media_type=None):
                                               if not any(a[config.ID_FIELD] == s[config.ID_FIELD] for s in subscribers)]
 
                 if len(subscribers_yet_to_receive) > 0:
+                    # Step 2(b)(iii)
+                    if updated.get('targeted_for'):
+                        subscribers_yet_to_receive = [s for s in subscribers_yet_to_receive
+                                                      if s.get('subscriber_type', '') != SUBSCRIBER_TYPES.DIGITAL]
```

Comment:
As per the ticket, `Any story restricted by state or region will automatically be restricted to internet clients.` That's the reason we can't restrict `Broadcast` subscribers here.